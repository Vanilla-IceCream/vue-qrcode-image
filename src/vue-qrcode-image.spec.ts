import { test, expect } from 'vitest';
import { mount } from '@vue/test-utils';

import Basic from './fixtures/Basic.vue';
import Size from './fixtures/Size.vue';

test('Basic', async () => {
  const wrapper = mount(Basic, {});
  expect(wrapper.html()).toMatchSnapshot();
});

test('Size', () => {
  const wrapper = mount(Size, {});
  expect(wrapper.html()).toMatchSnapshot();
});
