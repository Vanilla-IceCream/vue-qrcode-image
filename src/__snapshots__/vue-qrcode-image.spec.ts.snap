// Vitest Snapshot v1

exports[`Basic 1`] = `"<img src=\\"data:image/gif;base64,R0lGODdhSgBKAIAAAAAAAP///ywAAAAASgBKAAAC/4yPqcvtD6OctNqLs968+w+G4kiWZgKk6roqKfOiAMLWKmXb7rzENG/I1XDAHm94iPlkweKO6FCyfk3Y0cmcSFsBqYQLvoah3Wm5Ggl7y2Ot+HZOFrfwJRppf9DX6Ls5v3dFhvcW2LZGOPhXyHgoyFaHFSUp5+enNkfZl6YZh8jYtJRH5SYUaen5aKhTaqr6irqKJEL3Uxt3ovdmu4sb0haK5ZrJe0vEB7ljShpsXMG1eUkparY5qkv9lGxUTFzpZgUtB9grPgtuJB6sjdlt8Um9vB2/eD0p3U2InLP+DNcsjFU/ft/UYctkkOBAgansHYyWatI5hxiAFeyk0JcGi/CrmF20xhACx0beSO0LqQvfN4Ass5jsZEUlSEzk2MFskG1eyXGgNPr84m2ksp41L5wDKO9aopWtyA1D5s4XRZc85YWLBHUb0KFW02F9xKviv6qnpLJSOvZYwKX0LIZFx3XsURlJb8b0aq4a3rN2N+TV1u+lQQ9/qSbsCXQYKpCxwP48WJfxx4hm32HcKVMrWYj3cGJuiHBt2a2Qsr691XWqTbd0X8n6LJKo7JxsUcZuGSgwaHh6SS8FnOE3J9zT+kocPdwzUiELhfrr/PQrcY+JXS13GlSv6tZ1M+rOzTSX+PHky5s/jz69+vXs27s/UAAAOw==\\">"`;

exports[`Size 1`] = `"<img src=\\"data:image/gif;base64,R0lGODdhlACUAIAAAAAAAP///ywAAAAAlACUAAAC/4yPqcvtD6OctNqLs968+w+G4kiW5omm6sq27gvH8kzX9o3n+s73/g8MCofEovGIVAGWzKbzCYU0KdNHVQHNaq+hrTdqZVLF4SXjiyaL0mmpOsI9vxFsNKn+dZsn8UXfgOd1N8f3ludnSOjA9Qen2NGoB0AniLgH+Ch3GRBZNjnSuZi4pfmJuSlJiVq46hHawFiJNWrquPlaOngry0kLkqXqFBwMfIB7EUtq7PtRfCq8vKv8rFt7SB1dy+Hc+zSczX1skaz1je0K9kxOnN6t3Uwb/thILt+aUN8uXpHvnZ1ryRqvV/2gnesST9+8hdLK/fPEzt/BXwklugM4q6G9d/8ZBU67CKqiQZDqFAYyRzLltZQUNZr06LBkm4cHC64L6dKiTZErb9YcmGnDzoYRRwYiSlMl0HvbgopC+jOn0XYoxzAV6hQWs6gwN9LLihHh1acwUQ7typCjLbVNx2qFqlRqwbBW2WI9+tLs1rhlweEFi+yvzr0++eoVvK8u4r5cDzPmu7ha5KSFKxNeDNjG3ICbJybR0Bnfy8KfQYssFTpx6bDcirpOCmQj6qyTO84ebFHy3tdk8dL1i3ukbri83/79LXNqbpzK1bbmfBS5O6+ZIVGFHbB4KujLPXsXKxy7beB2x3d8/l31uOvfHe/7+rGqePDwy0OuXh89y/lt7Rf/JcieaFJpR556/ODHy232pRbgStUoFiCB5k13GnfhrYFgfBZul5yEFHYHnntp+UaefMlR51Z/It5T24eP/TdadRhYBmKLslmIon93PaafjXnhGGOKO+YXYYENBhmTgGgJuV6NCQLpZHAOUlakaVEmqWCPSGo5IHNEggjjlWfV5+FwL2bHnydTtnfDjWSZuNaa++ngpnFwSiKngSzUyZqMRl6JA59UzjRhn8T9eaF1wRlqB5pvDuocG2bCOWZaHBoGZaL9kQZpnrRdxqKkzK0YaagfyXknpCjQmOiXzemXamMnsEpmpmEu+OmhGMLFZalJopqmrEpYWuJ8AEr546PD/5q65JlsdohorHOWUCu06eX6arJ2rlCti5Rie5a0egam4XmY6WqcoM8qimWmJK5767dMZgDsiYhJG6+xfjbZrrnn4hutvvPWhee95P6qrYIPqmkwvzmKWyW94DZ84KkJbzgqw4LNuFRzBQ885MQWo1sHwNamACuirGZ5cW/7SgymyqD62lOGmnopMsIkiwpvsbvqaK+YEbNM6LQLS1CyrQe/a3TGSPOsZMounxRsiKw06u/NU+Nhcg+9Xlpmp0O3eaRbrl5abw5fQ4Sxsi6O68LabofNldQwY+aYdCur6ii15/JI7IR7XwuywxRzSjjgvAbu9798F/o4pomv+nK3jP+erbHHTlfsK9pla/32bvAUPu2xzSqcOeLscsys50Kb3fHRoaP7seLZFt01682qXjupt8rd+O6iP72oz7P32zQ6cg0f58yVAjorT84jebm2SZsQrtjIth5w3U/+nK/323fe/X3lWh207Q/3XfPyMSsvPO19w9A+0JyHn7ySNNQvO/9X7x+7zfmPePYzE+92pLvfRcd4A9Rf9Ej3PK4xMICQA9/BFEi1kzUwdxfsYPqYliPVwa15G5PZ+7zFvu/lb1OHW+DWqgc97P0Ng6ADnqdIB6ESfjBiNlThCFcDxCAKcYhELKIRj4jEJCpxiUxsohOfCMUoSnGKVKyiFa+IxSwDJqEAADs=\\">"`;
